website/src/components/subscriptions/overview/SubscriptionDisplay.tsx

```diff
@@ -73,7 +93,7 @@ function TriggerEvaluationResult({ result }: { result: Subscription['triggerEval
                     {result.type === 'ConditionMet' ? 'met' : 'not met'}
                 </span>
             </div>
-            <div className='text-gray-500'>Evaluated value: {result.evaluatedValue}</div>
+            <div className='text-gray-500'>Evaluated value: {roundToPrecision({ val: result.evaluatedValue })}</div>
```

Comment:
When I set the thershold to a value below 0.001, then I always get the "<" sign, even when the trigger condition is met.

I got two ideas, but I'm not sure how much effort they are in respect to the benefit:
1) Just show the value, rounded to the same precission as the thershold, when the evaluated value is above the thersohld.
2) Set the lower bound to 0.1% of the thershold, rounded to the next decimal. So a thershold of 0.0003 would have a lower bound of 0.0000001.

@chaoran-chen Does this even make sense to set that low thresholds?

Reply:
Difficult to say for sure but I can't see a good use case right way when a threshold below 0.1% would be useful, so I'm happy with defining this as the lowest allowed threshold.

Reply:
As discussed: We leave it like this. This means, that when the user inserts a very low threshold, then even when the evaluated value is larger than this threshold but still lower than the defined precision it is truncated to "<0.1%".
There are now two new tickets related: #244, #245.